examples/inference.ipynb

```diff
@@ -1879,6 +1901,262 @@
         "    print_prompt_and_generated(p, g)\n",
         "    print()"
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Perceiver AR symbolic audio modeling\n",
+        "\n",
+        "The following section demonstrates how to use a Perceiver AR audio model to generate symbolic ([MIDI](https://en.wikipedia.org/wiki/MIDI)) audio data. The model has `134M` parameters and was trained with an input context of `4096` prefix tokens and `2048` latent tokens for `27` epochs (`157M` tokens per epoch) on the [GiantMIDI-Piano](https://github.com/bytedance/GiantMIDI-Piano) dataset. \n",
+        "\n",
+        "The model accepts tokenized audio symbols extracted from [MIDI](https://en.wikipedia.org/wiki/MIDI) files as input. MIDI files are tokenized using the approach presented in the [Perceiver AR paper](https://arxiv.org/pdf/2202.07765.pdf). The tokenization method is described in detail in Section A.2 of [Huang et al (2019)](https://arxiv.org/abs/1809.04281) and the implementation used in this project is taken from the [Midi Neural Processor](https://github.com/jason9693/midi-neural-processor/blob/bea0dc612b7f687f964d0f6d54d1dbf117ae1307/processor.py).\n",
+        "\n",
+        "To generate audio sequences a 🤗 pipeline can be used. This pipeline is supplied with one or multiple MIDI files of type `pretty_midi.PrettyMIDI` acting as prompts. The pipeline returns the generated sequences either as `pretty_midi.PrettyMIDI` objects (using the option `render=False`) or as a `bytes` array containing the rendered [WAV](https://en.wikipedia.org/wiki/WAV) audio content (using `render=True`). Note that the utility [fluidsynth](https://www.fluidsynth.org/) must be installed on your system to render a MIDI file to the WAV audio format."
```

Comment:
Note that ~~the utility~~ fluidsynth must be installed ~~on your system~~ to render

(this should still be clear enough)